src/views/thread/container.js

```diff
@@ -445,8 +457,34 @@ class ThreadContainer extends React.Component<Props, State> {
               <Head
                 title={headTitle}
                 description={headDescription}
-                image={thread.community.profilePhoto}
-              />
+                type="article"
+                image={generateImageFromText({
+                  title: headTitle,
+                  footer: `spectrum.chat/${thread.community.slug}`,
+                })}
+              >
+                <meta name="twitter:card" content="summary_large_image" />
+                <meta
+                  property="article:published_time"
+                  content={new Date(thread.createdAt).toISOString()}
+                />
+                <meta
+                  property="article:modified_time"
+                  content={new Date(
+                    thread.modifiedAt || thread.createdAt
+                  ).toISOString()}
+                />
+                <meta
+                  property="article:author"
+                  content={`https://spectrum.chat/users/@${
```

Comment:
Why is the content a url and not just the author's name?

Reply:
Because that's how Open Graph works, not sure either. 🤷‍♂️ I researched it and people explicitly said this had to be URLs.